releases/v20.2.0-alpha.2.md

```diff
@@ -0,0 +1,350 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-alpha.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.2.0-alpha.2
+~~~
+
+### Backward-incompatible changes
+
+- The [`SHOW RANGE FOR ROW`](../v20.2/show-range-for-row.html) now takes a tuple of the row's index columns instead of the full column set of the row. [#50479][#50479]
+- Previously, issuing a `SIGTERM` signal twice or after another signal initiated a hard shutdown for a node. Now the first `SIGTERM` signal initiates a graceful shutdown and further occurrences of `SIGTERM` are ignored. To initiate a hard shutdown, issue `SIGINT` two times (or issue a `SIGINT` signal once after a `SIGTERM` signal). [#50539][#50539]
+- Specifying the same option multiple times in the `WITH` clause of the [`BACKUP`](../v20.2/backup.html) statement now results in an error message. Additionally, quoted option names are not allowed anymore. [#50723][#50723]
+
+### Security updates
+
+- HTTP endpoints beginning with `/debug/` now require a valid [admin](../v20.2/authorization.html) login session. [#50483][#50483]
+
+### General changes
+
+- Go 1.14 is now the minimum required version necessary to build CockroachDB. [#50671][#50671]
+- The docker image available with CockroachDB now includes the GEOS files required to work with Geospatial types in CockroachDB. [#50362][#50362]
+- Schema changes are now logged in greater detail. [#50372][#50372]
+- Added new options to YCSB to mirror OpenSource YCSB options. [#50546][#50546]
+- Links returned in error messages to point to unimplemented issues now use the CockroachLabs redirect/short-link server. [#49836][#49836]
+
+### Enterprise edition changes
+
+- The default flush interval for [changefeeds](../v20.2/change-data-capture.html) that do not specify a `RESOLVED` option is now 5s instead of 200ms to more gracefully handle higher-latency sinks. [#49770][#49770]
+- [Full cluster restore](../v20.2/restore.html#full-cluster) is now more resilient to transient transaction retry errors while it restores. [#49950][#49950]
+- [`RESTORE`](../v20.2/restore.html) now remaps user-defined types within the backup to existing types in the cluster. [#50349][#50349]
+- `BACKUP` can now run in `detached` mode, which means the Job ID is now returned immediately without waiting for the `BACKUP` job to finish. [#50775][#50775]
```

Comment:
Is "detached mode" a new thing?  If so it might be good to say something like "BACKUP can now run in a new 'detached' mode"